script/fix_aptc_pct_over_one_hundred_percent.rb

```diff
@@ -0,0 +1,19 @@
+require 'csv'
+
+report_name = "#{Rails.root}/enrollment_aptc_pct_list_#{TimeKeeper.date_of_record.strftime('%m_%d_%Y')}.csv"
+field_names = %w(
+        enrollment_hbx_id
+        faulty_aptc_pct
+        message
+      )
+
+CSV.open(report_name, "w", force_quotes: true) do |csv|
+  csv << field_names
+
+  affected_enrollments = HbxEnrollment.with_aptc.where(:created_at => {"$gte" => Date.new(2021,3,1)}).where(:elected_aptc_pct => {"$gt" => 1.0})
+  affected_enrollments.each do |enrollment|
+    enrollment.update_attributes!(elected_aptc_pct: 1.0)
```

Comment:
Should not hard code to 1.0  should be the code fix for ratio. Also, this should not only look for enrollments that have pct greater than 1 but also where the ratio is not equal to **cd_total_aptc / max_applicable_aptc**(from the code fix).